src/components/Onboarding/content.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react'
+import large1 from './highlights/assets/rinkeby/large/1.png'
+import large2 from './highlights/assets/rinkeby/large/2.png'
+import large3 from './highlights/assets/rinkeby/large/3.png'
+import large4 from './highlights/assets/rinkeby/large/4.png'
+import large5 from './highlights/assets/rinkeby/large/5.png'
+
+import small1 from './highlights/assets/rinkeby/small/1.png'
+// import small2 from './highlights/assets/rinkeby/small/2.png'
```

Comment:
Do we need to re-export this asset or is it missing?

Reply:
I removed the small assets as they were the same than the large ones https://github.com/aragon/court-dashboard/pull/76/commits/36a5d4440247b36b8a6bc9736c06ed1b65718868